### PR TITLE
feat(converter): add typeName for object generation

### PR DESCRIPTION
**Pull request by cakeinpanic**
Voila!

https://github.com/kreuzerk/svg-to-ts/issues/137

### CODE REVIEW

File: src/lib/options/default-options/default-object-conversion-options.ts [hunk 1]
```diff
@@ -7,5 +7,7 @@ export const DEFAULT_OBJECT_CONVERSION_OPTIONS: ObjectConversionOptions = {
   delimiter: Delimiter.CAMEL,
   verbose: false,
   fileName: 'my-icons',
-  objectName: 'icons'
+  objectName: 'icons',
+  generateType: true,
+  typeName: 'MyType'
```
Comment (kreuzerk) [inline src/lib/options/default-options/default-object-conversion-options.ts:12]: How about setting the default to `MyIconType`?

Comment (kreuzerk) [reply src/lib/options/default-options/default-object-conversion-options.ts:12]: We should also apply this default in the `commanline-object-options.collector.ts`. This file sets all the defaults for configurations that is read from a `.rc` config file

**Review by kreuzerk — CHANGES_REQUESTED** [review]
Hi @cakeinpanic thx so much for this PR. Awesome job! 👍 
I like the changes but still added two small comments. Let me know what you think. Also don't hesitate to reach out to me if you have any further questions.

File: src/lib/options/default-options/default-object-conversion-options.ts (hunk 1 above)
Comment (cakeinpanic) [reply src/lib/options/default-options/default-object-conversion-options.ts]: Fixed!
I did not pay attention that by mistake used `DEFAULT_CONST_CONVERSION_OPTIONS` where it was not meant to be

Comment (kreuzerk) [reply src/lib/options/default-options/default-object-conversion-options.ts]: Hi @cakeinpanic thx a lot. I checked the PR again and I think there is still one last thing missing. I think we also need to apply the default options in the `config-file-object-options.collector.ts`. Something like this.

`````

  if (!configOptions.generateType) {
    configOptions.generateType = DEFAULT_OBJECT_CONVERSION_OPTIONS.generateType;
    Logger.verboseInfo(`No 'generateType' property provided, "${DEFAULT_OBJECT_CONVERSION_OPTIONS.generateType}" will be used`);
  }

  if (!configOptions.typeName) {
    configOptions.generateType = DEFAULT_OBJECT_CONVERSION_OPTIONS.typeName;
    Logger.verboseInfo(`No 'typeName' property provided, "${DEFAULT_OBJECT_CONVERSION_OPTIONS.typeName}" will be used`);
  }
`````

Comment (cakeinpanic) [reply src/lib/options/default-options/default-object-conversion-options.ts]: @kreuzerk added them, also tackled readme

Comment (kreuzerk) [reply src/lib/options/default-options/default-object-conversion-options.ts]: Awesome 👍  Did you push the README? I don't see in in the change list

Comment (cakeinpanic) [reply src/lib/options/default-options/default-object-conversion-options.ts]: pushed now

**Comment by kreuzerk** [comment]
@all-contributors please add @cakeinpanic for code,doc,ideas

**Comment by kreuzerk** [comment]
https://github.com/all-contributors please add @cakeinpanic for code doc ideas

**Comment by kreuzerk** [comment]
@all-contributors please add @cakeinpanic for code doc ideas

**Comment by cakeinpanic** [comment]
@kreuzerk 🕺

**Pull request merged**